azure-pipelines.yml

```
@@ -1,6 +1,9 @@
 trigger:
   - develop
   - main
+  - 0.*-develop
+  - feature_dev_wip/partitioners
```

I assume the plan is to remove these feature branches when they are merged. Could these two lines be changed to the following pattern if we prefix all feature branches with `feature`?
```suggestion
  - feature*
```

---

Good catch -- these should be removed entirely

---

Updated.